.gitignore

```diff
@@ -52,6 +52,9 @@ qpixel-import.tar.gz
 # Ignore Vim stuff.
 *.swp
 
+# Ignore emacs stuff.
+*~
```

Comment:
Just so as we don't get bit in the rear down the line - are we sure there is no chance we get a file ending with a tilde at project root that's not related to emacs? LGTM otherwise

Reply:
Nvm, it's a standard pattern, all good